src/olympia/reviewers/templates/reviewers/review.html

```diff
@@ -245,18 +245,18 @@ <h3 id="history">
                   data-toggle-button-selector="#force_disable_addon"
                   id="force_enable_addon" type="button">{{ _('Force-enable add-on') }}</button>
         </li>
-
-        {% if not addon.block %}
         <li>
+        {% if addon.blocksubmission %}
+          <a href="{{ url('admin:blocklist_blocksubmission_change', addon.blocksubmission.id) }}" class="button"
+                  id="edit_addon_blocksubmission" type="button">{{ _('View Block Submission') }}</a>
```

Comment:
Just wondering why the `id` is `edit_addon_blocksubmission` but the text says "View Block Submission". Should it say "Edit Block Submission"?

Reply:
Technically it's a change page (edit) but depending on which permissions you have it would either be something you could change or something you could view and signoff.